tests/python_package_test/utils.py

```diff
@@ -0,0 +1,15 @@
+
+try:
```

Comment:
```suggestion
try:
```

Reply:
I didn't accept this change in the browser but I did make the change (removing the beginning blank line)